lib/LANraragi/Model/Tankoubon.pm

```diff
@@ -84,13 +101,59 @@ sub create_tankoubon ( $name, $tank_id ) {
 
     # Default values for new group
     # Score 0 will be reserved for the name of the tank
-    $redis->zadd( $tank_id, 0, redis_encode($name) );
+    $redis->zadd( $tank_id, $TANK_METADATA{"name"}, redis_encode("name_${name}") );
+    $redis->zadd( $tank_id, $TANK_METADATA{"summary"}, "summary_" );
+    $redis->zadd( $tank_id, $TANK_METADATA{"thumbhash"}, "thumbhash_" );
```

Comment:
Do those fields need default values? I'd just leave em as empty strings

Reply:
Here is the problem: Sorted sets require a unique key for the name, otherwise it overwrites the existent score, so, for example, if you have a Tank called "Name A"(Score 0), and then you add a summary field with the string "Name A"(Score -1), it will overwrite the first, so now your tank will not have score 0, hence unnamed, and this can happen to every field that the user can manually write, that's why I added a prefix to all metadata fields, so they are unique, even if they are blank, and I need them this way since I added a function to remove the prefix.

Reply:
I see, that makes sense. Let's keep the prefixes then I suppose 🤔

(I'm starting to wonder if we shouldn't just have used hashes here as well instead of trying to wrangle sorted sets)